File: Iniciante/Java/Iniciante.java
package Java;
import java.util.Scanner;
import java.lang.Math;
import java.text.DecimalFormat;

public class Iniciante{
    public static void main(String[] args){

        // 1000 - Hello World!
        System.out.println("Hello World!");

        // // 1001 - Extremamente Básico
        // Scanner J1001 = new Scanner(System.in);
        // int a = J1001.nextInt();
        // int b = J1001.nextInt();
        // J1001.close();
        // System.out.println("X = " + (a+b));

        // // 1002 - Área do Círculo
        // Scanner J1002 = new Scanner(System.in);
        // double raio = J1002.nextDouble();
        // J1002.close();
        // double area = 3.14159 * (Math.pow(raio, 2));
        // DecimalFormat df = new DecimalFormat("#.0000");
        // System.out.println("A=" + df.format(area));

        // // 1003 - Soma Simples
        // Scanner J1003 = new Scanner(System.in);
        // int a = J1003.nextInt();
        // int b = J1003.nextInt();
        // J1003.close();
        // System.out.println("SOMA = " + (a+b));

        // // 1004 - Produto simples
        // Scanner J1004 = new Scanner(System.in);
        // int a = J1004.nextInt();
        // int b = J1004.nextInt();
        // J1004.close();
        // System.out.println("PROD = " + (a*b));

        // 1005 - Média 1
        Scanner J1005 = new Scanner(System.in);
        double n1 = J1005.nextFloat();
        double n2 = J1005.nextFloat();
        J1005.close();
        double media = ((n1 * 3.5) + (n2 * 7.5))/11;
        DecimalFormat df1005 = new DecimalFormat("#.00000");
        System.out.println("MEDIA = " + df1005.format(media));

        // 1006 - Média 2
        
    }
}

File: Iniciante/Javascript/iniciante.js
// 1000 - Hello World!
console.log("Hello World!")

// 1001 - Extremamente Básico
function j1001(n1,n2){
    console.log(`X = ${n1+n2}`)
}

// 1002 - Área do Círculo 
function j1002(raio){
    let area = 3.14159 * (raio**2)
    console.log(`A=${area.toFixed(4)}`)
}

// 1003 - Soma Simples
function j1003(a,b) {
    console.log(`SOMA = ${a+b}`)
}

// 1004 - Produto simples
function j1004(x,y){
    console.log(`PROD = ${x*y}`)
}

// 1005 - Média 1
function j1005(n1,n2){
    let media = ((n1 * 3.5) + (n2 * 7.5))/11
    console.log(`MEDIA = ${media.toFixed(5)}`)
}

// 1006 - Média 2
function j1006(n1,n2,n3){
    let media = (n1 * 0.2) + (n2 * 0.3) + (n3 * 0.5)
    console.log(`MEDIA = ${media.toFixed(1)}`)
}

// 1007 - Diferença
function j1007(a,b,c,d){
    console.log(`DIFERENCA = ${(a*b)-(c*d)}`)
}

// 1008 - Salário
function j1008(n, h, s){
    console.log(`NUMBER = ${n}`)
    console.log(`SALARY = U$ ${(h*s).toFixed(2)}`)
}

// 1009 - Salário com bônus
function j1009(nome, sal, ven){
    let total = sal + (ven * 0.15)
    console.log(`TOTAL = R$ ${total.toFixed(2)}`)
}

// 1010 - Cálculo simples
function j1010(p1,p2){
    
}

// 1153 - Fatorial Simples
function j1153(n){
    let x = 1
    while (n > 0){
        x*=n
        n--
    }
    console.log(x)
}


